src/implementations/astroport/helpers.rs

```diff
@@ -92,18 +91,18 @@ pub(crate) fn adjust_precision(
 }
 
 /// ## Description
-/// Computes stable swap invariant (D)
+/// Computes the stableswap invariant (D).
 ///
 /// * **Equation**
 ///
 /// A * sum(x_i) * n**n + D = A * D * n**n + D**(n+1) / (n**n * prod(x_i))
 ///
 /// ## Params
-/// * **leverage** is the object of type [`u128`].
+/// * **leverage** is an object of type [`u128`].
```

Comment:
it says it is a u128 in the comment but in the function it is a `u64`. have we changed this or is the comment wrong in astro upstream aswell?

Reply:
Yeah the comment is wrong upstream. I removed it.